js/cards.js

```diff
@@ -107,32 +127,65 @@ function getOwnedCards(cards_data) {
   return ownedCards;
 }
 
-function sortCards(cards, sortType) {
+function sortCards(cards, sortType, reverse=false) {
   sortType = sortType ?? "Collector Number";
-  cards.sort(sort_functions[sortType]);
+  let before = reverse ? -1 : 1
+  let after = reverse ? 1 : -1
+  cards.sort(sort_functions[sortType](before, after));
   return cards;
 }
 
-export function showCollection(cards_data, htmlLocation) {
-  let sort = localStorage.getItem("sort");
+export function showCollection(cards_data, htmlLocation, page = 1) {
```

Comment:
this function is getting large, and the pagination button logic feels vulnerable to off by 1 bugs, but it works and I don't have any immediate suggestions to simplify things

Reply:
Glad to see I'm not the only one who saw it was getting too big. Time to make a separate js file for the collection! I hope it doesn't slow down development though, I'll try to get it done ASAP.

As for the button logic I do think working with 1-based indexing is weird and usually leads to trouble, but imo it works better in this case as it wouldn't make sense for a user to be in page 0. I can add more safeguards around stuff just to be safer, I'll take note of that.